### DISCUSSION TITLE
Journeys and Energy Consumption

### DISCUSSION THREAD

**Discussion by titvs** [tillsteinbach/VWsFriend#611] [Q&A] [answered]
Hi!
I have to congratulate Till for the work he have put into VWs Friend. Its a good way to keep track of our VWs data and keep a record.
Im writing concerning "Journeys" and its energy consumption calculation per 100 km. As far as ive understood, "Journeys" are a group of trips of which we want to know the statistics.
I have an ID3 (ver 3.2) which i have configured into VWs Friend (its total, usable battery capacity and WLTP), and ive been comparing the average consumption given by the car and calculated by VWs Friend. 
In a set of 3 trips (aggregated into a journey) with a total of 11 km, the car gave me an average of 12,4 kWh/100km. VWs Friend estimated 16.4 kWh/100km. As you can see its a steep difference so im asking if is possible to get out of VWs Friend better numbers since it seems it doesnt have access to the trip consumption provided by the car.
Thanks!

**Comment by tillsteinbach** [comment]
Interesting that VWsFriend seems to overestimate. Did you by chance check and compare the VW numbers with data from charging the vehicle?

**Comment by titvs** [comment]
Hi Till!
What do you mean? The SoC values? Im looking at a charging session from the sessions list and it records the dates and times, maximum power, start and end SoC and mileage when charging. The SoC that is displayed at VWs Friend is the same as the car's so thats working.
The other week i've noticed the same behaviour in a journey that consisted of 2 trips: 106 km total - VWs reported energy used of 18 kWh and an energy consumption of 17 kWh/100km. The car reported the same distance (106 km) but 15.3 kWh/100km. I guess you dont have access to the average trip consumption right?

These are the settings ive configured in VWs friend for my ID3 Pure Performance from 2021:
Usable Battery Capacity (net): 45 kWh
Total Battery Capacity (gross): 55 kWh
WLTP range: 349 km

I wonder how i can check this with the actual car values. I have and OBD scanner but ive found out that the SoC reported via the OBD isnt the same displayed by the car so it isnt very reliable...
Thanks!

**Reply by titvs** [reply]
@tillsteinbach , in another journey with a set of 2 trips (130 km total), VWs Friend reports 14.1 kWh/100km of energy consumption. The car reported 12.1 kWh/100km, which is around 12% less...

In the charging sessions, Ive now measured the actual energy charged with an energy meter and inserted this value into the "Real kWh charged" field of the corresponding charging session. I have now 4 charge sessions with these values.
If i go the "Charging Statistics" panel, VWs Friend reports the difference between estimated and real energy charged is at -5.63%. You think this is somehow related with VWs Friend overestimating the energy consumption?
Thanks!

**Comment by tillsteinbach** [comment]
No, -5.63% means that the value in VWsFriend is by 5.63% too low compared to what you actually charge. This is realistic because not all energy that goes in the car ends up in the battery. So I believe that the value in VWsFriend is not too low. Maybe the car is reporting the consumption too optimistic?

**Reply by titvs** [reply]
Well, the only way to verify this consumption difference is to verify the actual usable battery capacity of the vehicle. I dont know for sure if the values provided by Car Scanner via the OBD are reliable since the SOC value displayed by Car Scanner isnt the same as the SOC displayed by the car. I'll do some calculations taking into account the value "HV Battery energy content" reported by Car Scanner and see what total capacity i get.
Thanks Till.

**Reply by titvs** [reply]
So @tillsteinbach , ive been doing some calculations using data from an OBD II dongle and talked with other ID3 owners, and we came to the conclusion that using the SOC percentage to calculate the trip average isnt the ideal approach. The calculations taking into account the usable battery capacity provided by the OBD and the SOC percentages are never close the the value reported by the car. It seems it isnt linear. However if we use the difference of the value "HV battery energy content" provided by the OBD, at start and end, the average is similar to the onboard computer.
But even in this case, there seems to be situations with some sort of "adjustment" that takes place by the battery management system that doesn't reflect automatically by the values provided to the OBD.
So, regarding VWsF i guess you dont have access to the values of the actual battery energy content, do you? Or the calculated trip average? If not i guess there isnt a better way to calculate the average than the actual way you use...

**Reply by tillsteinbach** [reply, marked as answer]
I don't have access to the OBD values. For newer SW Versions I get trip data. I can try to match the trip data to my recordings, but I would need to find time to implement that. This will not happen soon.
